tests/test-headers.sh

```diff
@@ -18,7 +18,14 @@ INCLUDE_FILES="$(find "${SRCDIR}/include/" "${SRCDIR}/lib/" -name \*.h \
                       -a \! -name config.h            \
                       -a \! -name gettext.h)"
 
-TESTFILE="$(mktemp "${TMPDIR:-/tmp}/test-headers-XXXXXXXXXX.c")"
+# *BSD mktemp supports X's only at end of name
+TESTFILE="$(mktemp "${TMPDIR:-/tmp}/test-headers-XXXXXXXXXX")"
+if [ $? -ne 0 ]; then
+    echo "Could not create temporary file"
+    exit 1
+fi
+mv "$TESTFILE" "${TESTFILE}.c"
+TESTFILE="${TESTFILE}.c"
```

Comment:
nope, just "Create a temporary file or directory, safely, and print its name"

Reply:
Whatever hides behind that 'safely' ;-)
I would have assumed after creating the hash it would check for existence and loop back to hash-creation.
But I didn't check the implementation - and actually checking one might anyway not tell too much.

Reply:
Yep - that was my point. When you afterwards rename it it will possibly overwrite a pre-existing file reducing  mktemp to merely hash-generation. Was just brainstorming. As said above chances are low and this is test-code and won't run in production (just software production ;-) ).

Reply:
Ah, yeah I think we can ignore that